lib/api/core/models/repositories/repository.js

```diff
@@ -95,21 +96,33 @@ Repository.prototype.loadMultiFromDatabase = function (ids) {
 
   this.readEngine.mget(requestObject)
     .then(response => {
-      promises = [];
 
-      response.data.body.docs.forEach(document => {
-        var
-          object,
-          data;
+      if (!hydrate || !response.data || !response.data.body.docs) {
+        return deferred.resolve(
+          response.data.body.docs
```

Comment:
One of the possible ways to enter in this block statement is if `response.data` is null or undefined.
So, potentially, accessing to `response.data.body.docs` may generate a `TypeError` exception.